### DISCUSSION TITLE
Support request: autocomplete does not for justfiles and others

### DISCUSSION THREAD

**Discussion by igr** [aws/codewhisperer-command-line-discussions#79] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

When in a folder with `justfile`, typing `just` should open the popup with the receipts

### Actual behaviour

When I go to the very same project (or any other), and type just, the popup appears for a brief moment and then disappears. It is replaced with moving dots (like it is loading or something like that).

### Steps to reproduce

Use justfile.

**Comment by grant0417** [comment]
This seems to be an issue with the https://github.com/withfig/autocomplete spec, if you want to contribute a fix there we are happy to merge.

**Comment by brendanfalk** [comment]
Hey @igr realistically we are not going to be able to get around to prioritising this for at least the next month. Would you be able to investigate what's going wrong?

1. Here are the docs: fig.io/docs
2. Here is the just file spec: https://github.com/withfig/autocomplete/blob/master/src/just.ts

It should only take you about 5 minutes to learn how it all works and then hopefully you should be able to debug quickly

**Reply by igr** [reply]
I have understood fig is no longer supported, and this is its repo? Fig was working fine, aws not.
